### PR TITLE
updated synology-cloud-station (4.0-4203)

### PR DESCRIPTION
**Pull request by jpalmieri**

### CODE REVIEW

**Comment by jpalmieri** [comment]
I'm not sure why they changed the name from simply "Cloud Station" to "Cloud Station Drive" with this version but this is the same app, the OSX client for Synology Cloud Station.

**Comment by jpalmieri** [comment]
Also, this was created using `cask-repair`, so I wasn't able to change the cask name (it gave me an error when the cask name was different from the file name). If it's as simple as making sure the file name and cask name match, let me know and I'll change them to `synology-cloud-station-drive` along with the `name` key so there's parity all around.

**Comment by vitorgalvao** [comment]
> If it's as simple as making sure the file name and cask name match

It is. Would you rather I merge this and you submit another PR, or can you do it in this one?

**Comment by jpalmieri** [comment]
I'll take care of it in this one. Thanks!

**Comment by vitorgalvao** [comment]
Please @mention me after you do, so I get an email to check back.

**Comment by jpalmieri** [comment]
@vitorgalvao I decided to leave the filename and cask name as-is, since it will make upgrading this app easier.

Since you last saw this, I forced pushed a couple changes to the original commit:
* Updated the `name` key on line 6
* Updated the `pkg` path on line 10

File: Casks/synology-cloud-station.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-  url "https://global.download.synology.com/download/Tools/CloudStation/#{version}/Mac/Installer/synology-cloud-station-#{version.sub(%r{.*-}, '')}.dmg"
-  name 'Synology Cloud Station'
+  url "https://global.download.synology.com/download/Tools/CloudStationDrive/#{version}/Mac/Installer/synology-cloud-station-drive-#{version.sub(%r{.*-}, '')}.dmg"
+  name 'Synology Cloud Station Drive'
   homepage 'https://www.synology.com/'
   license :gratis
 
-  pkg "synology-cloud-station-#{version.sub(%r{.*-}, '')}.pkg"
+  pkg 'Install\ Cloud\ Station\ Drive.pkg'
```
Comment (vitorgalvao) [inline Casks/synology-cloud-station.rb]: Backslashes must be removed. The string is quoted.

**Comment by vitorgalvao** [comment]
> I decided to leave the filename and cask name as-is, since it will make upgrading this app easier.

We don’t support upgrade functionality.

Correctness is more important. Please change it.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 6822111cb7d837ce83526e90ab445d470041f2da. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by jpalmieri** [comment]
Thanks, @adidalal. I didn't see any difference with my changes and https://github.com/caskroom/homebrew-cask/commit/6822111cb7d837ce83526e90ab445d470041f2da. Just so I do it right next time, what did I miss?

**Comment by adidalal** [comment]
Squashed commits

**Comment by jpalmieri** [comment]
@adidalal :+1: thanks

**Pull request closed without merging**